infection.json.dist

```diff
@@ -7,5 +7,20 @@
     },
     "logs": {
         "text": "build/infection-log.txt"
+    },
+    "mutators": {
+        "@default": true,
+        "IdenticalEqual": false,
+        "NotIdenticalNotEqual": false,
+        "OneZeroInteger": {
+            "ignore": [
+                "Zalas\\Injector\\Service\\Injector::cannotHaveDuplicates"
```

Comment:
These two are ignored as the call to `cannotHaveDuplicates` doesn't technically affect the behaviour other than it makes its caller return earlier.